pep-0008.txt

```diff
@@ -1397,6 +1397,39 @@ annotations are changing.
   can be added in the form of comments.  See the relevant section of
   PEP 484 [6]_.
 
+Variable annotations
+--------------------
+
+PEP 526 introduced variable annotations. The style recommendations for them are
+similar to those on function annotations described above:
+
+- Annotations for module level variables, class and instance variables,
+  and local variables should have a single space after the colon.
+
+- There should be no space before the colon.
+
+- If an assignment has a right hand side, then the equality sign should have
+  exactly one space on both sides.
+
+- Yes::
```

Comment:
I like this style, but currently PEP 8 isn't explicit about function annotations styles (I could be missing it).  Rather than refer back to PEP 484 - which is large and covers many topics - maybe PEP 8 should be more explicit and prescriptive about both function and variable annotations?

Reply:
Actually PEP 8 has a small section on function annotations already: https://www.python.org/dev/peps/pep-0008/#function-annotations